libnetwork/netlabel/labels.go

```diff
@@ -39,6 +39,14 @@ const (
 	// DriverMTU constant represents the MTU size for the network driver
 	DriverMTU = DriverPrefix + ".mtu"
 
+	// AdvertiseAddrNMsgs is the number of unsolicited ARP/NA messages that will be sent to
+	// advertise an interface's IP and MAC addresses.
+	AdvertiseAddrNMsgs = Prefix + ".advertise_addr_nmsgs"
+
+	// AdvertiseAddrIntervalMs is the minimum interval in milliseconds between unsolicited
+	// ARP/NA messages sent to advertise an interface's IP and MAC addresses.
+	AdvertiseAddrIntervalMs = Prefix + ".advertise_addr_ms"
```

Comment:
Sure.

Reply:
Curious / silly rquestion; is this / will this be a user-facing option to configure? In general, I agree with @corhere's comment that using Go's accepted units is non-ambiguous. I'm slightly more on the fence for some user-facing options, or at least, we've been hit by some of those because;

- users now have to make themselves familiar with Go's accepted units / formats
- and/or we end up having to replicate those in our documentation (there's some existing cases where we fell into that; in some cases for longer durations, users expected, e.g. "if I can pass minutes, hours, why can't I use `days`??")
- somewhat related; users asking "why didn't you document `µs` or `ns`?", which _technically_ was supported, but for the purpose of that option very out of range (e.g. `--health-interval`)

So, I guess "it depends" a bit;

- Is this / will this be user-facing?
- What's the expected interval we can (and are willing to) support? (e.g. is `1µs` ok, is `48h` ok?) and do we have / do we need some guardrails around that?
- _if_ there's a reasonable / expected interval for these (e.g. we'd never expect / want to support anything outside of (e.g.) `1 ms - 1000 ms` (1 second) then _possibly_ being opinionated may be more clear (guidance on what kind of range we're expecting here); it's sure is more "limited", but could be less err-prone (of course, it doesn't defeat possible need for guardrails)

Reply:
Yes, it's user facing - but the default 1s should basically always be fine... it's configurable up to 2s because that's what the standards allow, and down to 100ms in case that's useful (we have no experience yet). It's always limited to up-to 3 messages, so it can't cause too much damage on a real network.

I also dislike the fancy units thing, having seen the issues it causes. Units in the param name make it unambiguous for users (rather than easier for coders). But, didn't feel too strongly in this case because it's very unlikely to be widely used.